### PR TITLE
Getting started page

### PR DESCRIPTION
**Pull request by Zokirkhon1002**

### CODE REVIEW

**Comment by Zokirkhon1002** [comment]
Translated Getting-started-page

File: content/community/nav.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- title: Community Resources
+- title: Hamjamiyat manbalari
   items:
     - id: support
       title: Support
```
Comment (doniyor2109) [inline content/community/nav.yml:4]: `title` ni ham tarjimaga qo'shsak bo'ladimi?

File: src/templates/components/NavigationFooter/NavigationFooter.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ const NavigationFooter = ({next, prev, location}) => {
                 textAlign: 'right',
               }}>
               <div>
-                <SecondaryLabel>Next article</SecondaryLabel>
```
Comment (doniyor2109) [inline src/templates/components/NavigationFooter/NavigationFooter.js:64]: `Previous article` ni ham tarjima qila olasizmi?
https://github.com/reactjs/uz.reactjs.org/pull/80/files#diff-9eb004a9d63a682485a443edf49a4b0363b96cb40bfed5423e1676223abecea5L43

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: src/templates/components/NavigationFooter/NavigationFooter.js (hunk 2 above)
Comment (Zokirkhon1002) [reply src/templates/components/NavigationFooter/NavigationFooter.js:64]: vaqt ajratishim bilan qilib qo'yaman

**Pull request closed without merging**